packages/desktop-client/src/components/transactions/SelectedTransactionsButton.tsx

```
@@ -145,6 +147,19 @@ export function SelectedTransactionsButton({
     return areNoReconciledTransactions && areAllSplitTransactions;
   }, [selectedIds, types, getTransaction]);
 
+  const canUnmarkTransactionsAsImported = useMemo(() => {
+    if (selectedIds.length === 0 || types.preview) {
+      return false;
+    }
+
+    const transactions = selectedIds.map(id => getTransaction(id));
+
+    const areSomeImportedTransactions = transactions.some(
+      tx => tx && tx.imported_payee,
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)